docs/slurm-perf-cluster.md

```diff
@@ -0,0 +1,183 @@
+High-Performance Multi-Node Cluster Deployment Guide
+===
+
+## Overview
+
+   This guide utilizes a collection of playbooks to configure a cluster, deploy a workload manager, and verify performance.
+
+   It leverages open source tools such as Pyxis and Enroot to optimize Slurm for multi-node Deep Learning jobs beyond the cluster configuration described in the [Slurm Deployment Guide](/docs/slurm-cluster.md). Additional details can be found [here](https://docs.nvidia.com/ngc/multi-node-bert-user-guide).
+
+## Supported Distributions
+
+   These packages have been installed and tested with the following Linux distributions and hardware platforms:
+
+   * [NVIDIA DGX OS 4.1 or greater](https://docs.nvidia.com/dgx/dgx-os-server-release-notes/index.html#dgx-os-server-sw-versions)
+   * DGX-1
+   * Non-blocking InfiniBand switches with DGX-1s [configured in InfiniBand mode](https://docs.nvidia.com/dgx/dgx1-user-guide/configuring-managing-dgx1.html#switching-from-ethernet-to-infiniband).
+   * Persistent network storage with [RAID cache](https://docs.nvidia.com/dgx/bp-dgx/storage.html#storage-nfs-cache-deep-learning) configured on DGX-1s
+
+## Requirements
+
+   * 1 or more DGX servers (Worker nodes)
+   * 1 Server or laptop (Ephemeral configuration/management machine)
+   * 1 Server or VM (Login node & Slurm controller)
+   * 1 or more NFS Servers
+   * Optional, 1 Server or VM (PXE provisioning machine)
+   * Internet access
+   * An ssh user that can be used to execute Slurm jobs, has sudo access, and can login to all hosts
+
+## Installation Steps
+
+1. Install a supported operating system on all nodes.
+
+   Install a supported operating system on all servers utilizing the [DGXie](/docs/dgxie-container.md) provisioning container, via a 3rd-party solution (i.e. [MAAS](https://maas.io/), [Foreman](https://www.theforeman.org/)), or server BMC/console.
+
+2. Set up your provisioning machine.
+
+   Install Ansible and required software on the provisioning machine.
+
+   DeepOps uses a single provisioning machine to deploy all other software to the cluster. This process may take several minutes as ansible-galaxy roles are downloaded and python packages are installed. For more information on Ansible and why we use it, consult the [Ansible Guide](ANSIBLE.md).
+
+   ```sh
+   # Install software prerequisites and copy default configuration
+   ./scripts/setup.sh
+   ```
+
+3. Edit the Ansible inventory and group_vars
+
+   Edit the Ansible inventory file and verify connectivity to all nodes.
+
+   Ansible uses an inventory which outlines the servers in the cluster and a set of group variables which playbooks use to customize deployment. The previous step created the `config` directory. It is also advised to copy over the slurm-perf example configuration.
+      
+   ```sh
+   # Copy the slurm-perf config
+   cp -rfp ./config/slurm-perf/ ./config
```

Comment:
Running this results in a non-zero error code. We should run `cp -rfp ./config/slurm-perf/* ./config` instead.

Reply:
Done